app/routes/application.js

```diff
@@ -82,20 +82,29 @@ export default Route.extend(ApplicationRouteMixin, {
       let hostImageTemplateUrl = get(show, 'about.people.firstObject.image.template');
       if (hostImageTemplateUrl) {
         stream.set('hostImageTemplateUrl', hostImageTemplateUrl);
+      } else {
+        stream.set('hostImageTemplateUrl', undefined);
       }
```

Comment:
does this end up any different than just calling
`stream.set('hostImageTemplateUrl', hostImageTemplateUrl);`
without the if block?

i guess `stream.set('hostImageTemplateUrl', hostImageTemplateUrl || undefined);` would be more exact if we want to be specific about handling other falsey values....